### PR TITLE
feat(fixtures): allow generating CVRs with ballot images

### PR DESCRIPTION
**Pull request by eventualbuddha**
## Overview

By default no ballot images will be included. Given `--includeBallotImages always` or `--includeBallotImages write-ins` then static ballot images will be included as data URLs for the appropriate records.

## Demo Video or Screenshot
```
❯ ./bin/generate-sample-cvr-file --includeBallotImages write-ins -e data/electionMinimalExhaustiveSample/electionMinimalExhaustiveSample.json
Wrote 168 cast vote records to output.jsonl
❯ cat output.jsonl | grep _ballotImages | head -n1 | cut -c -500
{"_ballotId":"id-4","_ballotType":"absentee","_precinctId":"precinct-1","_ballotStyleId":"1M","_testBallot":true,"_scannerId":"scanner","_batchId":"batch-1","_batchLabel":"Batch 1","best-animal-mammal":["otter"],"zoo-council-mammal":["zebra","lion","write-in-0"],"new-zoo-either":[],"new-zoo-pick":[],"_ballotImages":[{"normalized":"data:image/jpeg;base64,/9j/4AAQSkZJRgABAQEBLAEsAAD/2wBDAAYEBQYFBAYGBQYHBwYIChAKCgkJChQODwwQFxQYGBcUFhYaHSUfGhsjHBYWICwgIyYnKSopGR8tMC0oMCUoKSj/wAALCAzLCdQBAREA/8QAHwAA
```

## Testing Plan 
New automated tests, manually tested CLI.

## Checklist
- [ ] I have added [logging](https://github.com/votingworks/vxsuite/tree/main/libs/logging) where appropriate to any new user actions, system updates such as file reads or storage writes, or errors introduced.
- [ ] I have added JSDoc comments to any newly introduced exports

- [ ] I have added a screenshot and/or video to this PR to demo the change
- [ ] I have added the "user_facing_change" label to this PR to automate an announcement in #machine-product-updates

### CODE REVIEW

**Review by carolinemodic — APPROVED** [review]
I think the scan-service also exports layout information along with the image itself to help election-manager know how to crop things, but I know that the consuming code in election-manager isn't in main and might change.

**Comment by eventualbuddha** [comment]
This changed fairly substantially since your review, @carolinemodic. PTAL?

File: libs/fixtures/src/data/electionMinimalExhaustiveSample/ballot-package.zip.ts [hunk 1]
```diff
@@ -0,0 +1,50 @@
+/* Generated by res-to-ts. DO NOT EDIT */
```
Comment (carolinemodic) [inline libs/fixtures/src/data/electionMinimalExhaustiveSample/ballot-package.zip.ts:1]: Nice, I do think we should start just storing ballot packages for the elections in libs/fixtures for our own use

**Review by carolinemodic — APPROVED** [review]
Yay tests!

File: libs/cvr-fixture-generator/src/generate_cvrs.ts [hunk 2]
```diff
@@ -69,49 +72,69 @@ function getCandidateOptionsForContest(
...
+  scannerNames,
+  testMode,
+  includeBallotImages = 'never',
+}: {
+  ballotPackage: BallotPackage;
+  scannerNames: readonly string[];
+  testMode: boolean;
+  includeBallotImages?: IncludeBallotImagesOption;
+}): AsyncGenerator<CastVoteRecord> {
+  if (ballotPackage.ballots.some((b) => b.layout.length !== 2)) {
```
[the remark this answers is not in the document]
Comment (eventualbuddha) [reply libs/cvr-fixture-generator/src/generate_cvrs.ts:125]: Yes.

**Pull request merged**